client/signup/steps/plans/index.jsx

```diff
@@ -27,14 +27,35 @@ import { getSiteType } from 'state/signup/steps/site-type/selectors';
 import { getSiteTypePropertyValue } from 'lib/signup/site-type';
 import { saveSignupStep, submitSignupStep } from 'state/signup/progress/actions';
 import { recordTracksEvent } from 'state/analytics/actions';
+import { planItem } from 'lib/cart-values/cart-items';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
 export class PlansStep extends Component {
+	constructor( props ) {
```

Comment:
I don't like it any more than you do :)

I lifted the `skipRender` bit from the domains step.  The issue here is with how much of the signup code is contained in [step-actions](https://github.com/Automattic/wp-calypso/blob/a5e45b89533e6ef93a28dc9dde5a27b7576b26e8/client/lib/signup/step-actions.js) which are tightly coupled to the Signup framework. If we could just fire off redux actions to do these sorts of things, a lot of this could be handled at the controller level!